pipelines/collate_plots.py

```diff
@@ -294,37 +94,7 @@ def main(
     default=None,
 )
 
-parser.add_argument(
-    "--single-forecast-dir",
-    type=Path,
-    help="Path to a single directory to process",
-    default=None,
-)
-
-parser.add_argument(
-    "--disease",
-    type=str,
-    help="Name of the disease for which to collate plots.",
-    default=None,
-)
-
-parser.add_argument(
-    "--target-filenames",
-    type=str,
-    default=(
-        "Disease_forecast_plot.pdf Other_forecast_plot.pdf "
-        "prop_disease_ed_visits_forecast_plot.pdf "
-        "Disease_forecast_plot_log.pdf Other_forecast_plot_log.pdf "
-        "prop_disease_ed_visits_forecast_plot_log.pdf"
-    ),
-    help=(
-        "Exact filenames of PDF files to find and merge, including "
-        "the file extension but without the directory path, as "
-        "a whitespace-separated string"
-    ),
-)
-
 if __name__ == "__main__":
     args = parser.parse_args()
-    args.target_filenames = args.target_filenames.split()
-    main(**vars(args))
+    model_batch_dir = args.modelbatchdir
+    merge_and_save_pdfs(model_batch_dir)
```

Comment:
If you take the suggestion above.
```suggestion
    merge_and_save_pdfs(args.model_batch_dir)
```

Reply:
https://github.com/CDCgov/pyrenew-hew/pull/287/commits/43a15c4243329362c99cd5a1b34c6c66720e51ce